test/test-MetadataStore.js

```diff
@@ -181,6 +186,42 @@ exports.test_async_insert_all = function*(assert) {
   assert.deepEqual(items[7], [3, 5]);
 };
 
+exports.test_get_oldest_insert = function*(assert) {
+  // insert metadata at 3 different times
+  for (let metadata of metadataFixture) {
+    yield sleep(3000);
```

Comment:
Heh, agreed! Better specify the "created_at" manually to save us 9 seconds here 😃. Unfortunately, we need to change the function `getMetadataParameters` to get this field from the argument and use `Date.now()` if it's absent.